#24-Singly-linked-lists-and.md

```diff
@@ -0,0 +1,24 @@
+# Introduction to Singly Linked Lists and Doubly Linked Lists
+Singly linked lists and doubly linked lists are special types of data structure because they introduce the concept of pointers (links) and nodes. A linked list is a collection of similar items called nodes that seek to establish a sequence / order to the collection. There are two types of node -- the head and tail nodes and the head is always the first node.  Singly linked lists’ nodes can only access the next node, while doubly linked lists’ nodes can access both the previous and next node. The use of the term "singly" or "doubly" represent the number "links" that each node can establish with another node. Think of a line of lawn mowers strung together by rope.
+
+### Singly Linked Lists
+Singly linked lists’ nodes are connected only to the next node. Singly linked lists are great when you want to access data in the forward direction only. The insertion and deletion of nodes of nodes is easy. Singly linked list require less memory than doubly linked lists. However, the main problem with singly linked lists is that directly accessing the previous node is impossible. You experience examples of singly linked list when visiting a bank because the teller is typically required to process the customers in the order they arrive in. The problem with singly linked list becomes obvious when told to rewind a song but only the fast forward "button" works. 
```

Comment:
- change "lists'" to "list"
- reword "when you want to access data in the forward direction only" to "when you want to only access subsequent data"
- Why is insertion and deletion easy? is it because they require less memory?
- delete second "of nodes"
- make "list" plural
- delete the bank sentence
- reword last sentence, something like: "Imagine you're listening to an album, but the rewind button doesn't work. You can listen to the songs in sequential order, but you can't go back to listen to your favorite one again."